app/javascript/components/restaurants/Restaurant.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react'
+
+export class Restaurant extends React.Component {
```

Comment:
I just noticed you export everything twice. you can remove the export word here and just keep it at the end of the file where you actually do the exporting.